src/main/java/duke/MyTask/Task.java

```diff
@@ -0,0 +1,41 @@
+package duke.MyTask;
+
+public abstract class Task {
+    String name;
+    boolean isDone;
+    char c;
```

Comment:
Perhaps instead of naming the char 'c', could name it something else like taskType?